superset-frontend/src/SqlLab/components/SouthPane.jsx

```diff
@@ -59,6 +60,29 @@ const defaultProps = {
   offline: false,
 };
 
+const StyledPane = styled.div`
+  width: 100%;
+
+  .ant-tabs .ant-tabs-content-holder {
```

Comment:
This is probably not a battle for this PR, but when I see styles overriding AntD components, it makes me think there should probably be styling IN that component rather than here where it's used. In fact, this same sort of styling is happening in `superset-frontend/src/common/components/Tabs/Tabs.tsx` so there's probably some consolidation work to be done on this front.

Reply:
yeah, I agree. I felt that I was overwriting some styles for this component that were being used elsewhere but also didn't want to affect the other usages.. we can circle back on this it looks like.